.github/workflows/rust.yml

```
@@ -377,34 +379,47 @@ jobs:
     runs-on: ${{ matrix.os }}
     container: ${{ matrix.container }}
     steps:
-      - name: Checkout repository
+      - name: Checkout ockam cli repository
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
         with:
-          ref: ${{ github.event.inputs.release_branch }}
+          ref: ${{ inputs.ockam_command_cli_version != '' && inputs.ockam_command_cli_version || inputs.commit_sha  }}
+          path: ockam_cli
+
+      - name: Checkout ockam bats repository
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+        with:
+          ref: ${{ inputs.commit_sha }}
+          path: ockam_bats
 
       - name: Use cache cargo_home
-        uses: ./.github/actions/cargo_home_cache
+        uses: ./ockam_bats/.github/actions/cargo_home_cache
       - name: Use cache cargo_target
-        uses: ./.github/actions/cargo_target_dir_cache
+        uses: ./ockam_bats/.github/actions/cargo_target_dir_cache
+        with:
+          ockam_dir: ockam_cli
 
       - name: Build Binary
         shell: bash
+        working-directory: ockam_cli
         run: |
           rustc --version
           set -x
           cargo build --bin ockam
+          echo "PATH=$(pwd)/target/debug:$PATH" >> $GITHUB_ENV;
 
       - name: Run Script On Ubuntu
-        if: matrix.build == 'linux_86'
+        working-directory: ockam_bats
         shell: bash
         run: |
-          export PATH=$(pwd)/target/debug:$PATH;
+          ockam --version
```

```suggestion
          echo $(which ockam)
          ockam --version
```

Optional, but it may be nice to leave ourselves some breadcrumbs to quickly confirm that PATH is resolving the way you've designed it just above.

---

Done